system/lib/pthread/library_pthread_stub.c

```diff
@@ -277,6 +277,9 @@ int pthread_condattr_setpshared(pthread_condattr_t *attr, int shared) {
 }
 
 int pthread_getattr_np(pthread_t thread, pthread_attr_t *attr) {
+  attr->_a_stackaddr = emscripten_stack_get_base();
+  attr->_a_stacksize = emscripten_stack_get_base() - emscripten_stack_get_end();
+  attr->_a_guardsize = __default_guardsize;
```

Comment:
I wonder if we can just remove this stub and use `musl/src/thread/pthread_getattr_np.c` in the stub pthread library?

Reply:
Good idea, I did this.